common/src/mlflow_model_local_validation/run_model_sanity_validations.py

```diff
@@ -29,6 +30,75 @@
     handlers=handlers,
 )
 logger = logging.getLogger(__name__)
+tc = None
+
+
+def init_tc():
+    """Initialize telemetry client."""
+    global tc
+    if tc is None:
+        try:
+            tc = TelemetryClient("71b954a8-6b7d-43f5-986c-3d3a6605d803")
+        except Exception as e:
+            print(f"Exception while initializing app insights: {e}")
+
+
+def tc_log(message):
+    """Log message to app insights."""
+    global tc
+    try:
+        print(message)
+        tc.track_event(name="FM_import_pipeline_debug_logs", properties={"message": message})
```

Comment:
What's the reason to not use traces?

Reply:
Yes @skasturi. Discussed with Vikas, to remodel logging structure for all the components.